packages/docs/components/page-configs/va-option-list/api-options.ts

```diff
@@ -0,0 +1,23 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
Suggesting either remove or making more pleasing to eye.
![image](https://user-images.githubusercontent.com/5394573/82839638-b3823180-9ed8-11ea-9b5d-db4213b07950.png)